dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/config/IntegrationTestConfig.java

```diff
@@ -105,21 +103,4 @@ public DhisConfigurationProvider dhisConfigurationProvider()
 
         return dhisConfigurationProvider;
     }
-
-    private JdbcDatabaseContainer<?> initContainer()
-    {
-        // NOSONAR
-        DhisPostgreSQLContainer<?> postgisContainer = ((DhisPostgreSQLContainer<?>) new DhisPostgisContainerProvider()
-            .newInstance()) // NOSONAR
-                .appendCustomPostgresConfig( "max_locks_per_transaction=100" )
```

Comment:
Do we still have this config with the changed setup? Not sure why we are using it. Maybe a performance tuning thing for tests 🤔

Reply:
I think @ameenhere can help here
I can see from this testcontainers issue that this was a problem at some point https://github.com/testcontainers/testcontainers-java/issues/1278

What do we need this configuration for?
And do we have a way to configure it from JdbcUrl?

Reply:
I found the original PR that added it https://github.com/dhis2/dhis2-core/pull/2996
Since we got OOM errors back then we can still merge this PR and see if they reappear. The PR was from `Mar 13, 2019` so lots might have changed in the meantime 😋